frontend/my-study-mate/src/App.tsx

```
@@ -362,6 +362,24 @@ function App() {
               setTopics(topicsData);
               // Load saved expansions when topics are loaded from upload
               loadSavedExpansions(topicsData);
+              
+              // If topics are already loaded (from processed files), simulate processed chunks state
+              // so the "Generate Topics with AI" button becomes available
+              if (parsed.result.segments && Array.isArray(parsed.result.segments)) {
+                const totalWords = parsed.result.segments.reduce((sum: number, segment: { text: string }) => {
+                  if (segment && segment.text && typeof segment.text === 'string') {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)